test/UnitTestFramework/UnitTestFramework.js

```diff
@@ -1,357 +1,362 @@
-//-------------------------------------------------------------------------------------------------------

```

Comment:
Looks like this entire file was overwritten. Can you check your line-endings settings?